R/geom-text.R

```diff
@@ -89,6 +89,18 @@
 #'   scale_colour_discrete(l = 40)
 #' p + geom_label(aes(fill = factor(cyl)), colour = "white", fontface = "bold")
 #'
+#' # If border.color is NULL or not set, the border will use the text color
+#' p + geom_label(aes(color = factor(cyl)))
+#'
+#' # Alternatively, border.color can have a static value
+#' p + geom_label(aes(color = factor(cyl)), border.color = "black")
+#'
+#' # Use linetype and linewidth aesthetics to add highlights
+#' p + geom_label(
+#'   aes(fill = factor(cyl), linetype = qsec < 15),
+#'   border.colour = "black", color = "white", linewidth = 1) +
+#' scale_linetype_manual(values=c("solid", "blank"), limits = TRUE, labels = "1/4 mi < 15s", name = NULL)
```

Comment:
Some indentation and simplification because having clear examples is more important than having pretty examples :)
```suggestion
#' p + 
#'  geom_label(
#'    aes(fill = factor(cyl), linetype = qsec < 15),
#'    border.colour = "black", colour = "white"
#'  ) +
#'  scale_linetype_manual(values = c("solid", "blank"), limits = TRUE)
```

Reply:
Yeah, agreed!